equinix-openapi-metal/src/main/java/com/equinix/openapi/metal/v1/model/AbstractOpenApiSchema.java

```diff
@@ -46,7 +45,7 @@ public AbstractOpenApiSchema(String schemaType, Boolean isNullable) {
      *
      * @return an instance of the actual schema/object
      */
-    public abstract Map<String, GenericType> getSchemas();
+    public abstract Map<String, Class<?>> getSchemas();
```

Comment:
Here's a place where `import javax.ws.rs.core.GenericType` and it does require an additional code change, but even with this larger impact it's a small change.